modules/text-input/react/lib/TextInput.tsx

```diff
@@ -7,7 +7,7 @@ export interface TextInputProps
   extends GrowthBehavior,
     React.InputHTMLAttributes<HTMLInputElement> {
   error?: ErrorType;
-  inputRef?: React.RefObject<HTMLInputElement>;
+  inputRef?: React.Ref<HTMLInputElement>;
```

Comment:
Good catch! Was there a reason you only made this change to `TextInput`? There are several other components that use the `React.Ref` type.

Reply:
That's the only place it affected us 🤷‍♂

Reply:
Cool. Let's update the rest and merge this in for the next release

Reply:
I only found one other place where a ref that is a **prop exposed in the API** was typed `RefObject`. Added a commit that updates that instance.

Reply:
@davvidbaker good call on only worrying about the ones that are exposed. For Modal, a [PR recently when in that requires it to be a `RefObject`](https://github.com/Workday/canvas-kit/pull/328) . I've removed that commit and rebased, so we should be good to go now 👍